src/coreclr/jit/utils.cpp

```diff
@@ -453,6 +453,55 @@ void dspRegMask(regMaskTP regMask, size_t minSiz)
         regPrev = regNum;
     }
 
+#ifdef TARGET_XARCH
+    if (strlen(sep) > 0)
```

Comment:
can we convert this to a method and reuse the code with the `REG_FP_FIRST` case above?

Reply:
Sure thing. Going to handle this in a follow up PR since its for JIT Dump and not production code. We notably have quite places where JIT dump logic replicates code like this and it might be worth a longer term cleanup pass.

Reply:
Opened https://github.com/dotnet/runtime/pull/97459